pkg/resourcehandlers/github/gitinfo.go

```diff
@@ -24,7 +24,7 @@ func transform(commits []*github.RepositoryCommit) *git.GitInfo {
 			nonInternalCommits = append(nonInternalCommits, commit)
 		}
 	}
-	if len(commits) == 0 {
+	if len(nonInternalCommits) == 0 {
```

Comment:
It would be good if we check for both lengths of commits and nonInternalCommits

Reply:
done =>early exit